examples/annotation_types/tiled_imagery_basics.ipynb

```diff
@@ -0,0 +1,431 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">for polygon in pixel_polygons.__geo_interface__['coordinates']:
&nbsp; &nbsp; pixel_polygon = polygon[0]
&nbsp; &nbsp; points = [Point(x=x,y=y)for x,y in pixel_polygon]
&nbsp; &nbsp; points_in_4326 = [transformer(point) for point in points]
&nbsp; &nbsp; annotations.append(points_in_4326)

</pre><p>This whole section feels like we can make it more simple. All we are doing is projecting a shapely polygon.</p><p> </p><p>You can apply transformations to all points individually using shapely (so you don't need to know the geometry type).</p><p>https://shapely.readthedocs.io/en/stable/manual.html#other-transformations</p><p> </p><p>Maybe you can add a transform_shapely function. For now, we can leave that out of the scope but maybe make a ticket for this and get started on it after we merge this branch.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/389/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>